man/systemd.exec.xml

```diff
@@ -630,27 +630,23 @@
-        or <option>soft:hard</option> to set both limits individually (e.g. LimitAS=4G:16G).
-        Use the string <varname>infinity</varname> to
-        configure no limit on a specific resource. The multiplicative
-        suffixes K (=1024), M (=1024*1024) and so on for G, T, P and E
-        may be used for resource limits measured in bytes
-        (e.g. LimitAS=16G). For the limits referring to time values,
-        the usual time units ms, s, min, h and so on may be used (see
-        <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>
-        for details). Note that if no time unit is specified for
-        <varname>LimitCPU=</varname> the default unit of seconds is
-        implied, while for <varname>LimitRTTIME=</varname> the default
-        unit of microseconds is implied. Also, note that the effective
-        granularity of the limits might influence their
-        enforcement. For example, time limits specified for
-        <varname>LimitCPU=</varname> will be rounded up implicitly to
-        multiples of 1s.</para>
+        <listitem><para>These settings set both soft and hard limits of various resources for executed processes. See
+        <citerefentry><refentrytitle>setrlimit</refentrytitle><manvolnum>2</manvolnum></citerefentry> for details. The
+        resource limit is possible to specify in two formats, <option>value</option> to set soft and hard limits to the
+        same value, or <option>soft:hard</option> to set both limits individually (e.g. LimitAS=4G:16G).  Use the
+        string <varname>infinity</varname> to configure no limit on a specific resource. The multiplicative suffixes K
+        (=1024), M (=1024*1024) and so on for G, T, P and E may be used for resource limits measured in bytes
+        (e.g. LimitAS=16G). For the limits referring to time values, the usual time units ms, s, min, h and so on may
+        be used (see <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+        for details). Note that if no time unit is specified for <varname>LimitCPU=</varname> the default unit of
+        seconds is implied, while for <varname>LimitRTTIME=</varname> the default unit of microseconds is
+        implied. Also, note that the effective granularity of the limits might influence their enforcement. For
+        example, time limits specified for <varname>LimitCPU=</varname> will be rounded up implicitly to multiples of
+        1s. For <varname>LimitNICE=</varname> the value may be specified in two syntaxes: if prefixed with
+        <literal>+</literal> or <literal>-</literal>, the value is understood as regular Linux nice value in the range
+        -20..19. If not prefixed like this the value is understood as raw resource limit parameter in the range
+        1..40. As special quirk, the value 0 is also accepted even though it has the same effect as the value 1 or +19 (this is
```

Comment:
I wonder is this details needs to be explained in the man page... Maybe just say "in the range 0..40 (with 0 being equivalent to 1).".